product_docs/docs/pgd/5/routing/installing_proxy.mdx

```diff
@@ -53,9 +53,36 @@ You can set the log level for the PGD Proxy service using the top-level config p
 
 PGD Proxy uses endpoints given in the local config file only at proxy startup. After that, PGD Proxy retrieves the list of actual endpoints (route_dsn) from the PGD Proxy catalog. Therefore, the node option `route_dsn` must be set for each PGD Proxy node. See [route_dsn](../routing#configuration) for more information.
 
+##### Configuring health check
+
+PGD Proxy provides [HTTP(S) health check APIs](proxy#proxy-health-check). Add the below configuration parameters only if health check APIs are required. By default it is disabled.
+
+```yaml
+cluster:
+  name: cluster-name
+  endpoints:
+    - "host=bdr-a1 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a3 port=5432 dbname=bdrdb user=pgdproxy "
+    - "host=bdr-a2 port=5432 dbname=bdrdb user=pgdproxy "
+  proxy:
+    name: "proxy-a1"
+    endpoint: "host=proxy-a1 port=6432 dbname=bdrdb user=pgdproxy "
+	  http:
+  	    enable: true
+  	    host: "0.0.0.0"
+  	    port: 8080
+        secure: false
+        cert_file: ""
+        key_file: ""
+	    probes:
+  	      timeout: 10s
+```
+
+It can be enabled by adding the config `cluster.proxy.http.enable: true`. Default HTTP server address is `:8080`, timeout is `10s` and secure is `false`. To enable HTTPS set the config parameter `cluster.proxy.http.secure: true`. If it is set to `true`, the `cert_file` and `key_file` config parameters are mandatory. The `cluster.proxy.endpoint` parameter is only required for readiness API. It could be same as BDR node `dsn` with exception of host and port where host could be `localhost` or proxy hostname and port will be `listen_port` proxy option. User can add additional connection string parameters in this endpoint, like `sslmode`, `sslrootcert` or change DB user, if required.
```

Comment:
Explain what the health checks are *first* and what they need. Otherwise this section is meaningless to a new reader. 

The probes setting doesn't appear to be explained anywhere.

Especially, don't pack all this information into one paragraph.